### DISCUSSION TITLE
VideoStream works, trying to access Remote gives Hacker Alert. Invalid Request from IP...

### DISCUSSION THREAD

**Discussion by stansoltz** [GerryDazoo/Slinger#157] [Q&A] [answered]
My video stream works fine, but I can't seem to access the Remote.  the Server gives me this:

..Hacker Alert. Invalid Request from  ('184.151.XXX.XXX', 59641)
GET /remote HTTP/1.1
Host: 99.XXX.XX.XXX:8080
Connection: keep-alive
Cache-Control: max-age=0
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
Accept-Encoding: gzip, deflate
Accept-Language: en-US,en;q=0.9

My config file is as follows:

[SLINGBOX]
sbtype="350/500/M1/M2"
;sbtype="Solo/Pro/ProHD"
;sbtype="240"
password=hHXXXXXXXXXXi38
; uncomment the next lines ipaddress and port with your local network info
; if you don't want the server to automatically find your 
; slingbox on the local network. Needed if server and slingbox are
; not on the same LAN segment. Also removes requirement for the netifaces module  
ipaddress=192.168.1.145
port=5207
; Repace with your finderid, If and only if you need remote access and don't  
;have a static ip and you plan on using my service to access you server remotely. 
; Please read the release notes. Most people don't need this.
finderid=178BE0XXXXXXXXXXXXXXXXX43CAFF7

;If you've got more than one slingbox set name. THe server will use this when generating 
; logs to make it easier to see what's going on. 
;name=something

; Following are the default values
; Default resolution when server starts. You can change this and reconnect without having to restart the server
; valid range = 1..15
;Resolution=12

; Valid Values 1, 6, 10, 15, 20, 30, 60   depending on resolution
;FrameRate=30

;Min 50 Max 8000
;VideoBandwidth=2000

; Min 0 (auto) max 63
;VideoSmoothness=63

; Send Iframe every n seconds Max 30
;IframeRate=5

; Audio Bit Rate. Valid Options 16, 20, 32, 40, 48, 64, 96 Default 32
;AudioBitRate=32

; Video Source  0, 1, 2, 3 depending on your hardware corrosponds to one of
;Composite, Component, HDMI or Tuner.
;ProHD 0=Tuner, 1=composite 2=svideo 3=component
;Solo, M1 0=composite 1=Svideo 2=component
;500  0=composite, 1=component, 2=HDMI
; If you don't set this the code will use the last configired input
; I recommend setting this value. If not set correctly you'll often only see a black screen. 
VideoSource = 1

[SERVER]
; local port number for the server to listen on for connections
port=8080
;maxstreams=10
enableremote=yes

[REMOTE]
; see release notes for changing this if your remote doesn't work, But as a start make it 
;the same as your configured VideoSource 0-3. See above.
code=1
; path to an external remote control definitions file, if the defaults
; aren't working for you or you want to change the format of the web page.
; use the supplied remote.txt as a starting point.
include=remote.txt

When I start the Server, I get this:

Version : 3.08 Running on Windows-10-10.0.19043-SP0 pid= 7996
Using config file config.ini
Connection Manager Running on port 8080 with 10 max streams using URL slingbox.
Streamer Running:  10 config.ini SLINGBOX 8080 8080 8388608
starting up on  port 8080 8388608
Checking for slingbox at ('192.168.1.145', 5207)
('192.168.1.145', 5207) OK
8080 Using slingbox at  ('192.168.1.145', 5207)
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Reading Custom Remote definition from remote.txt
 * Serving Flask app 'slingbox_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
10/20/2022, 13:51:29  Registering Slingboxes {'178BE048XXXXXXXXXXXXXD743CAFF7': 8080}
Getting external IP address
Redirector, my external IP address 99.XXX.XX.XXX
Registering 178BXXXXXXXXXXXXXXXCAFF7:8080

When I start the Video Stream I get:

Register Successful
10/20/2022, 14:54:47   Streaming connection from ('184.XXX.XXX.XXX', 59716)
Starting remote stream 1

Slinginfo  "350/500/M1/M2" 12 30 192.168.1.145 5207 3072 10
8080 Starting Stream for  184.XXX.XXX.XXX:59716
Connecting... ('192.168.1.145', 5207) Control
8080 Selecting Video Source 1
Keycodes= [1, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 22, 24, 25, 27, 28, 29, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 46, 72, 73, 74, 75, 76, 77]
VideoParameters: Resolution= 12 FrameRate= 30 VideoBandwidth= 2000 VideoSmoothness= 63 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.1.145', 5207) Stream
8080 Stream started at 10/20/2022, 14:54:50  744 0
.

Any Idea why I get the Hacker Alert when I try to connect to the remote?
Thanks

**Comment by GerryDazoo** [comment, marked as answer]
You're spelling remote wrong.  Remote

**Reply by stansoltz** [reply]
Sorry, where exactly do you see it misspelled?  Or, do you mean when I enter the URL in a browser the "R" in Remote should be capitalized.

**Reply by stansoltz** [reply]
Thank you, GerryDazoo!  Your work to save my slingboxes is truly appreciated.  I'm sure I'll have more questions, but, yes, the capital R in Remote was the trick.  Perhaps add that to your README.txt so you don't have to deal with silly questions like this.  I'm thinking of making a screen capture youtube video to help other Slingboxers out!  Cheers from Canada!
